lib/Controller/RecoveryController.php

```diff
@@ -89,12 +89,12 @@ public function adminRecovery($recoveryPassword, $confirmPassword, $adminEnableR
 				Http::STATUS_BAD_REQUEST);
 		}
 
-		if (isset($adminEnableRecovery) && $adminEnableRecovery === '1') {
+		if ($adminEnableRecovery !== null && $adminEnableRecovery === '1') {
```

Comment:
```suggestion
		if ($adminEnableRecovery === '1') {
```
The way it is now looks a bit silly - if `$adminEnableRecovery === '1'` then it is sure to be not null.